### ISSUE TITLE
Cannot login to GooglePlay ( server says: BadAuthentication )

### ISSUE THREAD

**Issue by aleixsb** [matlink/gplaycli#34]
I'm trying to use this project, I've configured the credentials.conf with my google acc, but i'm getting the following error...

Cannot login to GooglePlay ( server says: BadAuthentication )

I'm doing something wrong or google has changed anything?

Thanks!

**Comment by rena2019** [comment]
same here

**Comment by craftyguy** [comment]
If you have two-step auth. enabled in your google account, you have to create an application password for gplaycli. Do you have two-step auth. enabled?

**Comment by Der4NDR3** [comment]
Hello,
I've got the same problem (two-step auth. was already disabled). 
I used my real device ID and a new PlayStore test account. After I changed the credentials in the original credentials.conf ( _/home/<userprofile>/.config/gplaycli/credentials.conf_ ), I get the message:
_Cannot login to GooglePlay ( server says: UNKNOWN_ERR )_
Then I switch to my real PlayStore account (where the used device id was never connected to, but some other devices) and now I can connect!

I think the new PlayStore account was the problem. Maybe it's a useful information for you
